dwave/embedding/utils.py

```diff
@@ -230,3 +230,95 @@ def edgelist_to_adjacency(edgelist):
         else:
             adjacency[v] = {u}
     return adjacency
+
+def adjacency_to_edgelist(adjacency):
```

Comment:
I think this is a misleadingly named function, because it actually returns an iterator, not a list. See comments on `chain_edges`.